pkgs/development/cuda-modules/_cuda/db/schema.nix

```diff
@@ -0,0 +1,608 @@
+            };
+            isJetson = mkOption {
+              type = bool;
+              default = false;
+              description = ''
+                Marks a feature set corresponding to Jetson embedded devices.
+
+                Corresponds to the `"linux-aarch64"` system string in NVIDIA manifests, as distinguished from e.g. `"linux-sbsa"` (binaries for `aarch64-linux` hosts with external PCI-e GPUs).
+              '';
+            };
+            isArchitectureSpecific = mkOption {
+              type = bool;
+              default = lib.hasSuffix "a" name;
+              defaultText = literalExpression ''hasSuffix "a" ${name}'';
+              description = ''
+                Marks a feature set implemented by a physical architecture. "Architecture-specific" compute capabilities are neither forward- nor backward-compatible.
+              '';
+            };
+            isFamilySpecific = mkOption {
+              type = bool;
+              default = lib.hasSuffix "f" name;
+              defaultText = literalExpression ''hasSuffix "f" ${name}'';
+              description = ''
+                Marks a feature set implemented by a family of architectures, e.g. `"10.0f"` for "compute capabilities supported by and only by Blackwell Data Center GPUs".
+              '';
+            };
+            minCudaMajorMinorVersion = mkOption {
+              type = types.nonEmptyStr;
+              description = ''
+                First CUDAToolkit release featuring binaries for this compute capability.
+              '';
+            };
+            maxCudaMajorMinorVersion = mkOption {
+              type = nullOr types.nonEmptyStr;
+              description = ''
+                Last CUDAToolkit release featuring binaries for this compute capability.
+              '';
+            };
+            dontDefaultAfterCudaMajorMinorVersion = mkOption {
+              type = nullOr types.nonEmptyStr;
+              description = ''
+                CUDAToolkit version after which Nixpkgs no longer includes this capability in default targets. Used to infer defaults compatible with special products such as TensorRT.
+              '';
+            };
+          };
+        }
+      );
+    in
+    {
+      cudaCapabilityToInfo = mkOption {
+        type = attrsOf CapabilityInfo;
+        description = ''
+          `∷ CapabilityName ⇒ CapabilityInfo`
+
+          Elaborates Nixpkgs cuda capability strings.
+        '';
+        # example = bootstrapData.cudaCapabilityToInfo;
+      };
+      cudaArchNameToCapabilities =
+        let
+          computeDefault = db: allCCs: lib.groupBy (cc: db.cudaCapabilityToInfo.${cc}.archName) allCCs;
+        in
+        mkOption {
+          type = attrsOf (types.nonEmptyListOf types.nonEmptyStr);
+          description = ''
+            `∷ CudaFamilyName ⇒ List CapabilityName`
+
+            Shorthand for compilation targets corresponding to each CUDA architecture family.
+          '';
```

Comment:
Worth noting that these cannot be blindly used together -- belonging to the same family doesn't mean capabilities can be built together (https://developer.nvidia.com/blog/nvidia-blackwell-and-nvidia-cuda-12-9-introduce-family-specific-architecture-features/).